src/index.js

```diff
@@ -36,6 +36,11 @@ const parseFromStd = (stdout, stderr) => {
   return parsed;
 };
 
+const getProjectDirectory = () => {
```

Comment:
This will _always_ return the first project, regardless of where the current file is actually located. As discussed in the comments this should take a `filePath` parameter and either just use `atom.project.relativizePath(filePath)[0]` or you will need to parse through the paths returned from `getDirectories` and find the one that actually contains the current file.

Note that taking a `filePath` parameter means you can remove the logic on each call that you have currently.